account/accounts.go

```diff
@@ -365,6 +370,55 @@ func (m *Manager) Accounts() ([]gethcommon.Address, error) {
 	return filtered, nil
 }
 
+// StartOnboarding starts the onboarding process generating accountsCount accounts and returns a slice of OnboardingAccount.
+func (m *Manager) StartOnboarding(accountsCount, mnemonicPhraseLength int) ([]*OnboardingAccount, error) {
+	m.mu.Lock()
+	defer m.mu.Unlock()
+
+	onboarding, err := NewOnboarding(accountsCount, mnemonicPhraseLength)
+	if err != nil {
+		return nil, err
+	}
+
+	m.onboarding = onboarding
+
+	return m.onboarding.Accounts(), nil
+}
+
+// RemoveOnboarding reset the current onboarding struct setting it to nil and deleting the accounts from memory.
+func (m *Manager) RemoveOnboarding() {
+	m.mu.Lock()
+	defer m.mu.Unlock()
+
+	m.onboarding = nil
+}
+
+// ImportOnboardingAccount imports the account specified by id and encrypts it with password.
+func (m *Manager) ImportOnboardingAccount(id string, password string) (Info, string, error) {
+	var info Info
+
+	m.mu.Lock()
+	defer m.mu.Unlock()
+
+	if m.onboarding == nil {
+		return info, "", ErrOnboardingNotStarted
+	}
+
+	acc, err := m.onboarding.Account(id)
+	if err != nil {
+		return info, "", err
+	}
+
+	info, err = m.RecoverAccount(password, acc.mnemonic)
+	if err != nil {
+		return info, "", err
+	}
+
+	m.onboarding = nil
```

Comment:
Why do we reset it?

Reply:
@adambabik just to remove everything related to the keys and the keys themselves